new-docs/getting-started/core_concepts.mdx

```diff
@@ -0,0 +1,11 @@
+---
+title: SQLite Cloud - Core Concepts
+description: Learn about the core concepts behind SQLite Cloud.
+slug: /core-concepts-slug
+---
+
+**Outline**
+- Nodes, Node Types, and Clusters -> link to /scaling
+- 
```

Comment:
@marcobambini Could you add any other core concepts you think users may find useful and may be unfamiliar with?